front-end/components/appInputComponents/constants.tsx

```diff
@@ -16,6 +16,7 @@ export const COMPONENT_TYPES = [
   "RangeSlider",
   "ColorPicker",
   "DateTimePicker",
+  "ChatTextField",
```

Comment:
Don't need it here. These components are just for the input components that are embedded inside the app